app/models/user.rb

```diff
@@ -7,14 +7,15 @@ class User < ApplicationRecord
   has_one_attached :photo
   has_one :coach_profile, dependent: :destroy
   has_one :sport, through: :coach_profile
-  has_many :lessons_to_attend, class_name: 'Lesson', foreign_key: 'student_id', inverse_of: :student
-  has_many :lessons_to_teach, class_name: 'Lesson', foreign_key: 'coach_id', inverse_of: :coach
+  # has_many :lessons_to_attend, class_name: 'Lesson', foreign_key: 'student_id', inverse_of: :student
+  # has_many :lessons_to_teach, class_name: 'Lesson', foreign_key: 'coach_id', inverse_of: :coach
   has_many :students, through: :lessons
```

Comment:
don't think association is required here.

Reply:
Thanks! I did all the changes except this 1... this provides us a shortcut to use User.lessons